easybuild/easyconfigs/o/OpenKIM/make-kim-model-list.py

```diff
@@ -0,0 +1,32 @@
+from urllib.request import urlopen
```

Comment:
@schiotz Please include license header + mention authorship (so people can contact you in case of questions/problems concerning this script)